### ISSUE TITLE
Run parallel jobs for different loss functions

### ISSUE THREAD

**Issue by tanzir5** [odissei-lifecourse/life-sequencing-dutch#13]
I have a few different pretraining tasks that can be incorporated with MLM loss objective. We can run parallel jobs for pretraining using different loss functions. The objective is to gain understanding into which tasks are good for the model to pretrain on. 

While the multi-gpu thing is being fixed, this can be done in parallel.

**Comment by tanzir5** [comment]
Moved to #149
